docs/API.md

```diff
@@ -64,6 +64,13 @@ protocol stream.
     to the element containing the remote session screen. The default value is `rgb(40, 40, 40)`
     (solid gray color).
 
+`qualityLevel`
+  - Is an `int` in range `[0-9]` controlling [`JPEG Quality Level Pseudo-encoding`](https://github.com/rfbproto/rfbproto/blob/master/rfbproto.rst#jpeg-quality-level-pseudo-encoding). Value `0` is equivalent to
```

Comment:
Can we describe this more in terms a user might understand? They shouldn't have to know the details of the protocol.

Reply:
Would this be too much of a simplification?

`qualityLevel`
  - Is an `int` in range `[0-9]` controlling the desired JPEG quality. Value `0` implies low quality and `9` implies high quality.
    Default value is `6`.

Reply:
That sounds like a reasonable level to me.

Reply:
Ok, fixed then.